Source/ACE.Server/WorldObjects/Creature_Magic.cs

```diff
@@ -98,42 +98,33 @@ public void CreateCreatureSpell(uint spellId)
 
         public uint CalculateManaUsage(Creature caster, SpellBase spell, WorldObject target = null)
         {
-            var items = new List<WorldObject>();
+            uint mana_conversion_skill = caster.GetCreatureSkill(Skill.ManaConversion).Current;
+            uint difficulty = Math.Max(25, spell.Power);     // for mana conversion only?
```

Comment:
Spells are not clamped to a base difficulty of 25; see http://asheron.wikia.com/wiki/Creature_Enchantment_Mastery_(Spell)

Reply:
see discord conversation